kaldo/observables/harmonic_with_q.py

```diff
@@ -302,24 +298,26 @@ def calculate_eigensystem_unfolded(self, only_eigenvals=False):
         fc_s = fc_s.reshape((n_unit_cell, 3, supercell[0], supercell[1], supercell[2], n_unit_cell, 3))
         supercell_positions = self.second.supercell_positions
         supercell_norms = 1 / 2 * np.linalg.norm(supercell_positions, axis=1) ** 2
-        dyn_s = np.zeros((n_unit_cell, 3, n_unit_cell, 3), dtype=np.complex)
+        dyn_s = np.zeros((n_unit_cell, 3, n_unit_cell, 3), dtype=complex)
         supercell_replicas = self.second.supercell_replicas
         for ind in range(supercell_replicas.shape[0]):
             supercell_replica = supercell_replicas[ind]
-            replica_position = np.tensordot(supercell_replica, cell, (-1, 0))
+            replica_position = np.tensordot(supercell_replica, cell.T, (-1, 0))
             distance = replica_position[None, None, :] + (atoms.positions[:, None, :] - atoms.positions[None, :, :])
```

Comment:
Nice catch, can you replace it with 
`replica_position = np.tensordot(supercell_replica, cell, (-1, -1))`
or
`replica_position = np.tensordot(supercell_replica, cell, (-1, 1))`
instead, please? 
This way you don't need to calculate the transpose at every step.